x-pack/solutions/observability/plugins/infra/public/components/kubernetes_dashboard_promotion/kubernetes_dashboard_promotion.tsx

```diff
@@ -0,0 +1,318 @@
+          'xpack.infra.kubernetesDashboardPromotion.ecs.integrationDocsLinkText',
+          {
+            defaultMessage: 'Kubernetes Integration',
+          }
+        ),
+        testSubj: 'infraKubernetesDashboardCardIntegrationDocsLink',
+      },
+    },
+    actionButton: {
+      installed: {
+        dashboardsPath: '#/list?_g=()&s=tag:(Kubernetes)',
+        label: i18n.translate('xpack.infra.kubernetesDashboardPromotion.ecs.viewDashboardsLink', {
+          defaultMessage: 'View Dashboards',
+        }),
+        testSubj: 'infraKubernetesDashboardCardLink',
+      },
+      notInstalled: {
+        integrationPath: '/detail/kubernetes',
+        label: i18n.translate('xpack.infra.kubernetesDashboardPromotion.ecs.viewIntegrationLink', {
+          defaultMessage: 'View Integration',
+        }),
+        testSubj: 'infraKubernetesDashboardCardInstallLink',
+      },
+    },
+    hideButton: {
+      label: i18n.translate('xpack.infra.kubernetesDashboardPromotion.ecs.hideCard', {
+        defaultMessage: 'Hide this',
+      }),
+      ariaLabel: i18n.translate('xpack.infra.kubernetesDashboardPromotion.ecs.dismissCard', {
+        defaultMessage: 'Dismiss card',
+      }),
+      testSubj: 'infraKubernetesDashboardCardHideThisButton',
+    },
+  },
+  link: {
+    dashboardsPath: '#/list?_g=()&s=tag:(Kubernetes)',
+    label: i18n.translate('xpack.infra.kubernetesDashboardPromotion.ecs.bottomDrawerLink', {
+      defaultMessage: 'Kubernetes Integration',
+    }),
+    testSubj: 'inventory-kubernetesDashboard-link',
+  },
+};
+
+const OTEL_CONFIG: IntegrationConfig = {
+  card: {
+    imageType: 'otel',
+    title: {
+      installed: i18n.translate(
+        'xpack.infra.kubernetesDashboardPromotion.otel.integrationInstalledTitle',
+        { defaultMessage: 'View Kubernetes OpenTelemetry Dashboards' }
+      ),
+      notInstalled: i18n.translate(
+        'xpack.infra.kubernetesDashboardPromotion.otel.integrationNotInstalledTitle',
+        { defaultMessage: 'Install Kubernetes OpenTelemetry Dashboards' }
+      ),
+    },
+    description: {
+      installed: (integrationLink: React.ReactNode) => (
+        <FormattedMessage
+          id="xpack.infra.kubernetesDashboardPromotion.otel.integrationInstalledDescription"
+          defaultMessage="View dashboards available for the Kubernetes clusters which are observed using {integrationLink} which match your query."
+          values={{ integrationLink }}
+        />
+      ),
+      notInstalled: (integrationLink: React.ReactNode) => (
+        <FormattedMessage
+          id="xpack.infra.kubernetesDashboardPromotion.otel.integrationNotInstalledDescription"
+          defaultMessage="We have dashboards for Kubernetes clusters which are observed using {integrationLink} which match your query."
+          values={{ integrationLink }}
+        />
+      ),
+      docsLink: {
+        url: 'https://www.elastic.co/docs/reference/integrations/kubernetes_otel',
```

Comment:
Should we hardcode these docLink URLs? If so, could we centralize them in constants somewhere?

Perhaps the constante part could be `https://www.elastic.co/docs/reference/integrations/`, and the package name (also a constant) be appended to the URL